src/components/tools/table-tool/table-tool.tsx

```diff
@@ -129,17 +228,19 @@ export default class TableToolComponent extends BaseComponent<IProps, IState> {
     return [x, y];
   }
 
-  private getTableActionLinks(): ILinkProperties {
-    const actionId = uniqueId();
+  private getTableActionLinks(): ILinkProperties | undefined {
```

Comment:
I can't tell what caused these changes - were there difficulties getting the feature to work with linked graphs?

Reply:
Technically, they're unrelated. I just realized that there's no reason to include link properties if the table isn't linked.